modules/ROOT/pages/using-policies.adoc

```diff
@@ -1,13 +1,13 @@
-= Applying a Policy
+= Apply a Policy
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, endpoint
 
 
-By default, a policy applies to the entire API. 
+You can apply a policy to the entire API (default) or to specific methods and resources of an API. 
 
-== Task Prerequisites
+== Prerequisites
 
 Before you apply a default policy, ensure that:
 
```

Comment:
Hidden text: wasn't sure if you wanted me to edit it?

* Mule runtime engine (Mule) is installed with xref:api-gateway-capabilities-mule4.adoc[API gateway capabilities] enabled.
* An xref:api-instance-landing-page.adoc[API] configured with a basic endpoint or a proxy endpoint exists in API Manager.
[[c/b "API Manager contains an API configured with ...."]]
* A Mule application is deployed with a xref:connectors::http/http-listener-ref.adoc[HTTP(s) based flow], which is linked to the API through Autodiscovery.
[[ correct to "...with an HTTP-based or HTTPS-based flow that is linked to the API through autodiscovery."]]
+
When Autodiscovery is correctly configured in your Mule application, your application’s API is said to be tracked by or paired to API Manager.
[[autodiscovery is lowercase. Also, why "said to be" instead of just "is"?]]
Autogenerated proxies handle these prerequisites in most cases. API gateway reconciles the automated policies applied to an API in API Manager during runtime.
[[ GLOBALLY try to replace "handle," per Style Ref. 
Add "The" to get "The API gateway reconciles...."]]
== Apply a Policy
To apply a policy:
* Go to *API Manager > API Administration* and select the API instance from the details window.
* From the left navigation, select *Policies > Apply New Policy*.
[[end with colon instead of period ^]]
+
image::applying-default-policies.png[75%,75%]
* Expand the policy that you want to apply.
* Select the policy version and click *Configure Policy*.
+
For details about the configuration parameters for each of the policies, see the specific xref:policies-ootb-landing-page.adoc[policy documentation].

Reply:
Ignore extra bullets ^

Reply:
Done! Thanks!